platform/mbed_lib.json

```diff
@@ -70,6 +70,31 @@
             "help": "Enable tracing of each memory call by invoking a callback on each memory operation. See mbed_mem_trace.h in the HAL API for more information",
             "value": false
         },
+        "sys-stats-enabled": {
+            "macro_name": "MBED_SYS_STATS_ENABLED",
```

Comment:
Setting this as macro name will set the macro? Wow I didn't knew we can actually rename long `MBED_CONF` parameters using this..

Reply:
Yeah it's super helpful but not a really well known feature.